### PR TITLE
Update google analytics tag

### PR DESCRIPTION
**Pull request by echappen**
# Pull request summary
Updates Google Analytics tag to the [provided tag](https://github.com/18F/TLC-crew/issues/586)

Resolves https://github.com/18F/TLC-crew/issues/586 for 18f.gsa.gov

## Reminder - please do the following before assigning reviewer

- [ ] update readme - NA
- [ ] For frontend changes, ensure design review - NA

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introdcued

### CODE REVIEW

File: _includes/analytics.html [hunk 1]
```diff
@@ -1,14 +1,6 @@
 {% if site.google_analytics_ua %}
```
Comment (cantsin) [inline _includes/analytics.html:1]: We should also use `site.google_analytics_ua` setting directly here.

File: _includes/analytics.html [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
 <!-- Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics_ua }}"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag() { dataLayer.push(arguments); }
-  gtag('js', new Date());
-
-  // `forceSSL` was used for analytics.js (the older Google Analytics script). It isn't documented for gtag.js, but the term occurs in the gtag.js code; figure it doesn't hurt to leave it in. -@afeld, 5/29/19
-  gtag('config', '{{ site.google_analytics_ua }}', { 'anonymize_ip': true, 'forceSSL': true });
```
Comment (cantsin) [inline _includes/analytics.html:10]: Should we keep `anonymize_ip` or not here?

**Review by cantsin — COMMENTED** [review]
Quick review notes

**Review by cannandev — APPROVED** [review]
LGTM. Are the two items in the checklist done?

**Pull request merged**